pkg/apis/admissionregistration/types.go

```diff
@@ -165,6 +165,40 @@ type ExternalAdmissionHook struct {
 	// allowed values are Ignore or Fail. Defaults to Ignore.
 	// +optional
 	FailurePolicy *FailurePolicyType
+
+	// Decides whether to run the webhook on an object based on whether the
+	// namespace for that object matches the selector. If the object itself is a
+	// namespace, the matching is performed on object.metadata.labels.
+	//
+	// For example, to run the webhook on any objects whose namespace is not
+	// associated with "runlevel" of "0" or "1";  you will set the selector as
+	// follows:
```

Comment:
Please document giving sample JSON or yaml, as the consumers of this documentation won't necessarily look at go source code.